cpp/src/io/parquet/bloom_filter_reader.cu

```diff
@@ -584,8 +584,6 @@ std::optional<std::vector<std::vector<size_type>>> aggregate_reader_metadata::ap
                                             stream);
 }
 
-equality_literals_collector::equality_literals_collector() = default;
```

Comment:
why is this deleted?

Reply:
Moved to the header where it's declared.